modules/services/sharing/src/main/kotlin/au/com/shiftyjelly/pocketcasts/sharing/SharingClient.kt

```diff
@@ -412,6 +435,23 @@ data class SharingRequest internal constructor(
         ) : Data {
             override fun toString() = "ClipVideo(title=${episode.title}, uuid=${episode.uuid}, start=${range.start.toSecondsWithSingleMilli()}, end=${range.end.toSecondsWithSingleMilli()})"
         }
+
+        class WebLink internal constructor(
```

Comment:
This might be too generic and lead to some handling edge cases in the future. WDYT about making it a `ReferralLink` and handling all text properties inside? In the constructor we would then pass only the referral code.

Reply:
I agree, refactored it in 7245975